build.md

```diff
@@ -57,6 +57,28 @@ Install the build dependencies.
   * Build the binary.
   * Build the docker image with the binary.
 
+* To build multi-arch images, you need to have `buildx` installed. Follow these [instructions](https://github.com/docker/buildx#installing) to install buildx.
```

Comment:
Rename the file as BUILD.md

Reply:
Also, can you push the Git Development workflow into another document named "git-workflow.md"

Reply:
Done